index.html

```diff
@@ -0,0 +1,327 @@
+            <h3>Prayer</h3>
+            <p>Heartfelt prayer.</p>
+          </div>
+          <div
+            class="
+              d-flex
+              my-1
+              event-bg
+              justify-content-around justify-content-md-center
+              align-items-center
+              flex-md-column
+              col-12
+              text-center
+            "
+          >
+            <img src="img/icons/worship-icon.png" alt="worship" />
+            <h3>Worship</h3>
+            <p>Worship in truth and spirit.</p>
+          </div>
+        </div>
+        <div
+          class="see-height d-flex justify-content-center align-items-center"
+        >
+          <button class="d-lg-none bg-join">
+            Join L2 Ministries Conference 2021
+          </button>
+          <p class="under-p d-none d-lg-block">SEE THE WHOLE PROGRAM</p>
+        </div>
+      </section>
+    </div>
+
+    <div class="container">
+      <section class="speakers">
+        <div class="my-5">
+
+       
+        <h3 class="text-center">Featured Speakers</h3>
+        <div class=" underline">
+          <hr >
+        </div>
+      </div>
+        <div class="row">
+          <section class="col-md-6 d-flex f-speaker mb-5">
+            <div class="img">
+
+            </div>
+            <div class="d-flex flex-column  ps-3">
+              <h3></h3>
+              <p></p>
+              <p></p>
+            </div>
+            
+          </section>
+          <section class="col-md-6 d-flex   f-speaker mb-5">
+             <div class="img">
+              
+            </div>
+            <div class="d-flex flex-column  ps-3">
+              <h3></h3>
+              <p></p>
+              <p></p>
+            </div>
+          </section>
+        </div>
+        <div class="row">
+          <section class="col-md-6 d-flex  f-speaker mb-5">
+            <div class="img">
+              
+            </div>
+            <div class="d-flex flex-column ps-3">
+              <h3></h3>
+              <p></p>
+              <p></p>
+            </div>
+          </section>
+          <section class="col-md-6 d-flex  f-speaker mb-5">
+            <div class="img">
+              
+            </div>
+            <div class="d-flex flex-column  ps-3">
+              <h3></h3>
+              <p></p>
+              <p></p>
+            </div>
+          </section>
+        </div>
+      </section>
+    </div>
+    <footer>
+      <div class="container-fluid footer-bg ">
+          <section class="container d-flex flex-column justify-content-center align-items-center">
+              <div class="partner-header d-flex flex-column justify-content-center align-items-center">
+  
+          
+  <h4>Partner</h4>
+  <div class=" underline">
+      <hr >
+    </div>
+   
+  </div>
+  <ul class="d-flex list-unstyled flex-wrap justify-content-around col-12">
+    <li>Petra Ministries</li>
+    <li>Mustard Seed Unilab </li>
+    <li>TACSFON UI</li>
+  </ul>
```

Comment:
Partner section is missing logos

- [ ] Kindly update this section
_see reference_
![image](https://user-images.githubusercontent.com/67957533/132745036-8b4bbd02-b280-4ba2-ad98-af48b94cf793.png)

Reply:
I believe you were referring to this and which I used the hx tags as logos, This is also here on my page.

![I believe you were referring to this](https://user-images.githubusercontent.com/65824881/132749637-961254a2-a12c-40da-a0ca-5956807b43bc.png)